libs/react-helpers/src/lib/generic-data-provider.ts

```diff
@@ -646,7 +646,7 @@ function makeDerivedDataProviderInternal<
         const updatedPart = parts[updatedPartIndex];
         if (updatedPart.isUpdate) {
           isUpdate = true;
-          if (updatedPart.delta && combineDelta && data) {
```

Comment:
**nitpick(non-blocking):** this part is hard to evaluate without a test covering it, and obviously the change hasn't broken a test. It feels quite fragile.

I'll hit Approve but I feel like I'm taking your word for it :crossed_fingers:

Reply:
Condition with check if delta exists was not tested. It's removed so don't need to check it,